src/Components/Intro/Intro.js

```diff
@@ -1,40 +1,54 @@
 import React from "react";
 import AnchorLink from 'react-anchor-link-smooth-scroll';
 
+const ciphers = [
+    "atbash",
+    "caesar",
+    "vigenere"
+]
+const cipherPretty = [
+    "Atbash",
+    "Caesar",
+    "Vigenere"
+]
+
 function Intro(props) {
     return (
         <div id="intro">
             <div className="container">
-                <div className="column">
+            <div className="columns is-vcentered container">
+                <div className="column is-9">
                     <div className="columns is-vcentered container">
-                        <div className="is-size-3 content-custom column container">
-                        <br />
-                        What is encryption? 
+                        <br/>
+                        <div className="is-size-4 content-custom column container">
+                        
                         How does ZHOFRPH WR JUDYLWB IDOOV turn into WELCOME TO GRAVITY FALLS? 
                         And how is that related to computers??  
-                        </div>
-                    </div>
-                    <div className="columns is-vcentered container">
-                        <div className="is-size-3 content-custom column container">
+                        <br/><br/>
+                        <span role="img" aria-label="key">🔑</span>   <span role="img" aria-label="computer">💻</span>   <span role="img" aria-label="keywithlock">🔐</span>
+                        <br/><br/>
                         Computers use cryptography all the time. 
                         Humankind has encrypted messages since as early as 1500 B.C. 
                         Let’s start from the beginning... 
-                        <br /><br />
                         </div>
-                    </div> 
+                    </div>
+                </div>
+                <div className="column"> 
+                    {ciphers.map((cipher, i) => {
+                        return(
+                            <div>
+                                <br/>
+                                <AnchorLink offset={-5} href={"#"+cipher}>
+                                    <button class="button is-static is-large is-family-secondary has-text-weight-bold">{cipherPretty[i]} Cipher</button>
+                                </AnchorLink>
+                                <br/><br/><br/>
+                            </div>
+                        )
+                    })}

```

Comment:
I noticed you used a lot of line breaks here to get the desired look. This works visually, but can be dangerous on account of the fact that your actual anchor link getting pushed up or down the page from where it appears. Instead, try something kind of like this:

```suggestion
                    {ciphers.map((cipher, i) => {
                        return(
                            <button class="button is-static is-large is-family-secondary has-text-weight-bold">
                                <AnchorLink offset={-5} href={`#${cipher}`}>
                                    {cipherPretty[i]} Cipher
                                </AnchorLink>
                            </button>
                        )
                    })}

```